package.json

```diff
@@ -9,7 +9,7 @@
     "build-analyze": "NODE_ENV=production npm run build -- --buildtype production --analyzer; npm run analyze",
     "prebuild": "node script/prebuild.js",
     "heroku-serve": "http-server",
-    "heroku-postbuild": "npm run build --  --entry static-pages,style",
+    "heroku-postbuild": "sh script/heroku-postbuild.sh",
```

Comment:
Add a separate file, so that we can clearly see the logic for install the `vagov-content` sibling repo.